src/events.rs

```diff
@@ -145,6 +147,22 @@ macro_rules! make_pointer_event {
     };
 }
 
+macro_rules! make_touch_event {
+    ($name:ident, $type:literal) => {
+        make_event!($name, $type => web_sys::TouchEvent);
+
+        impl $name {
+            #[inline] pub fn ctrl_key(&self) -> bool { self.event.ctrl_key() || self.event.meta_key() }
```

Comment:
Man the DOM sucks, duplicating all these methods. Oh well.

Reply:
Indeed :/ Do you have some improvements in mind?

Reply:
Unfortunately no, the best we can do is use a macro to remove the duplication, but that doesn't really solve it. It's just a bad design in the DOM.